src/integrationtests/java/com/aws/greengrass/integrationtests/deployment/DeploymentConfigMergingTest.java

```diff
@@ -709,6 +742,145 @@ void GIVEN_kernel_running_service_WHEN_run_with_change_THEN_service_restarts() t
             }
         }
     }
+
+    @Test
+    void GIVEN_kernel_running_with_some_config_WHEN_merge_deployment_config_successful_THEN_config_is_updated()
```

Comment:
why do we need this test? What is it covering which isn't already covered?

Reply:
discussed offline, existing tests use offline configuration which skips connectivity validation code